deployment/provision_vagrant_vm.sh

```diff
@@ -100,7 +106,7 @@ apt-get -q install -y libasound2 libgconf-2-4 libgbm1 libgtk-3-0 libnss3 libx11-
 wget https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh --no-verbose --output-document - | sudo -H -u $USER bash
 
 # setup evap
-cd "$MOUNTPOINT"
+cd "$MOUNTPOINT" || exit 1;
```

Comment:
To solve the second part of https://github.com/e-valuation/EvaP/pull/2105/files#r1488291372, this should be
```suggestion
cd "$REPO_FOLDER" || exit 1;
```
Not `$MOUNTPOINT` but `$REPO_FOLDER` is affected by bindfs.